fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'develop'
+# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', ref: 'develop'
```

Comment:
@AliSoftware Is this syntax correct, or should it have been reverted back to `branch`? Since it's commented out, I don't think we have to deal with this right now and start another set of CI tasks that'd delay the code freeze, but I thought I'd point it out in case it's wrong.

Reply:
Oh good point. And good question.

I think it _might_ still work – because a branch in git is also a ref so if you reference a commit by its (branch or tag) name instead of sha1 it probably works as well? But haven't tested, and it's more consistent to have it use `branch:` anyway. I'll quick-fix it in the release-branch post-code-freeze to be sure to not forget, thanks for flagging!

Reply:
Although technically branches are also refs, they use a naming pattern of `refs/heads/{branch_name}`. So, I think it'd depend on how it's implemented in Bundler(?). While working on the automated gutenberg-mobile PRs project, I found that this was not very consistent across GitHub API & libraries I worked with, so I wouldn't be surprised either way.

Reply:
FIxed in 8dfe547a6dbd33af7b8e0cac08799cb9cb4c74e1